projects/storefrontapp-e2e-cypress/cypress/integration/regression/site-context/language/language-my-account-pages.e2e-spec.ts

```diff
@@ -103,7 +106,7 @@ describe('Language switch - my-account pages', () => {
     it('should change language in the page', () => {
       siteContextSelector.siteContextChange(
         personalDetailsPath,
-        siteContextSelector.LANGUAGES,
+        [siteContextSelector.LANGUAGES, siteContextSelector.TITLES],
```

Comment:
Tested it just passing `siteContextSelector.TITLES` and it seems to pass every time.

So maybe we can remove `siteContextSelector.LANGUAGES` and just past `siteContextSelector.TITLES`.
It make the code more readable and be consistent with what we had before.